fastlane/Fastfile

```diff
@@ -120,14 +147,22 @@ lane :open_pr_upgrading_dependencies do |options|
       set_upstream: true,
       remote_branch: branch
     )
+
+    message = "[AUTOMATIC] #{message_ios}#{message_android}".strip
+
+    labels = ["dependencies"]
+    if type_of_bump_ios == :minor || type_of_bump_android == :minor
```

Comment:
Hmm just to check, if it's a major are we still creating the automatic PRs? And if so, should we also add a `major` label here?

Reply:
We are skipping if it's not within the same major, that's why I didn't bother checking here. We do that in `update_native_dependencies_to_latest`, where we only get the latest release within the same major